test/specs/rules/require-trailing-comma.js

```diff
@@ -35,6 +35,22 @@ describe('rules/require-trailing-comma', function() {
         output: 'var t = {\n\ta: 1,\n\tb: 2, };'
     });
 
+    reportAndFix({
+        name: 'missing comma in object pattern',
+        rules: rules,
+        errors: 1,
+        input: 'const { foo, bar } = baz;',
+        output: 'const { foo, bar, } = baz;'
+    });
+
+    reportAndFix({
+        name: 'missing comma in array pattern',
+        rules: rules,
+        errors: 1,
+        input: 'const [ foo, bar ] = baz;',
```

Comment:
Yeah this one is for `require-trailing-comma` since I figure we might as well update both - the other one is [here](https://github.com/jscs-dev/node-jscs/pull/1891/files#diff-e5d250618fad9c3add6b8a0bb15d51c6R51)